sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogV2Util.scala

```diff
@@ -296,6 +297,49 @@ private[sql] object CatalogV2Util {
     }
   }
 
+  /**
+   * Apply Table Constraints changes to an existing table and return the result.
+   */
+  def applyConstraintChanges(
```

Comment:
this is not apply, but more like collect constraints.

Reply:
good point. renamed